app/views/auth/login.haml

```diff
@@ -0,0 +1,136 @@
+        vertical-align: baseline;
+      }
+      /* HTML5 display-role reset for older browsers */
+      article, aside, details, figcaption, figure,
+      footer, header, hgroup, menu, nav, section {
+        display: block;
+      }
+      a:link, a:visited {
+        color: #0093b0;
+        text-decoration: none;
+        display: block;
+      }
+      a:active, a:hover {
+        color: #e86e2f;
+      }
+      body {
+        line-height: 1;
+        background: #fee9aa;
+        color: #3f3f3f;
+        font: 100% lato, verdana, arial, helvetica, sans-serif;
+      }
+      #page-wrap {
+        margin: 2em auto;
+        width: 22em;
+      }
+      #login-form {
+        background: #fff;
+        border-radius: .75em;
+        margin-bottom: 1em;
+        padding: 1.25em 1.5em 2em;
+        position: relative;
+        box-shadow: 0 2px 2px rgba(51, 51, 51, .3);
+      }
+      img {
+        display: block;
+        height: auto;
+        margin: 0 auto .75em;
+        width: 180px;
+      }
+      input[type=email], input[type=password], input[type=text] {
+        border: solid 1px #bbb;
+        border-radius: .25em;
+        font-size: 1em;
+        height: 2em;
+        padding: .25em .5em;
+        width: 17.85em;
+        -webkit-appearance: none;
+        appearance: none;
+      }
+      .with-errors input {
+        border: solid 1px #860024;
+      }
+      input#submit {
+        background: #eb8723;
+        border: none;
+        color: #fff;
+        cursor: pointer;
+        display: block;
+        font-family: lato, arial, helvetica, sans-serif;
+        font-size: 1em;
+        margin: 0;
+        padding: .5em .625em;
+        text-align: center;
+        text-decoration: none;
+        width: 9em;
+        -moz-border-radius: .1875em;
+        -webkit-border-radius: .1875em;
+        border-radius: .1875em;
+        box-shadow: 0 3px 0 #e16a3e;
+        background: -moz-linear-gradient(top, rgb(244, 182, 38) 0%, rgb(235, 135, 35) 100%);
+        background: -ms-linear-gradient(top, rgb(244, 182, 38) 0%, rgb(235, 135, 35) 100%);
+        background: -o-linear-gradient(top, rgb(244, 182, 38) 0%, rgb(235, 135, 35) 100%);
+        background: -webkit-linear-gradient(top, rgb(244, 182, 38) 0%, rgb(235, 135, 35) 100%);
+        background: linear-gradient(top, rgb(244, 182, 38) 0%, rgb(235, 135, 35) 100%);
+      }
+      p { font-size: 1em; margin-bottom: 1em; }
+      p#forgot-login {
+        bottom: 1.375em;
+        font-size: .8em;
+        line-height: 140%;
+        position: absolute;
+        right: 2em;
+        text-align: right;
+      }
+      .error-message {
+        color: #860024;
+      }
+    %body
+    #page-wrap
+      = form_tag session_path(:user), :id => "login-form", :class => @error.present? ? "with-errors" : "" do
+        = image_tag "cc-logo.png", :title => "The Concord Consortium", :alt => "The Concord Consortium"
+        %p
+          Log in with your Learn Portal account
```

Comment:
This code will get applied to other portals besides the learn.concord.org portal. So you should use the site_name from settings.yml and that should work.